### ISSUE TITLE
Session filter in standard theme "indico style"

### ISSUE THREAD

**Issue by VBoudry** [indico/indico#5577]
Could a filter be added to the "indico style" theme (view=standard), similar to what exist in the default theme ?
That would render the indico style more usable for events with parallel sessions.

Have a nice day,
Vincent Boudry.

**Comment by ThiefMaster** [comment]
![image](https://user-images.githubusercontent.com/179599/203277186-037bbc6f-adda-4c8e-bd9c-ce52175f5278.png)

Try this button in the top bar ;)

**Comment by VBoudry** [comment]
I didn't notice this button appearing in the standard view.

Now on this page (as example) 
https://indico.in2p3.fr/event/27968/timetable/?view=standard,
the application of a filter switches back to the normal view:
https://indico.in2p3.fr/event/27968/timetable/?showDate=2022-11-22&showSession=20#20221122
instead of
https://indico.in2p3.fr/event/27968/timetable/?view=standard&showDate=2022-11-22&showSession=20#20221122
which is what I need.

But at least I have a way, now, thank you !
Have a nice day.

Adrian a écrit le 22/11/2022 à 10:28 :

**Comment by ThiefMaster** [comment]
Good point. Maybe we should fix this behavior when using a non-standard theme and preserve the `view` arg...

**Comment by VBoudry** [comment]
An additional comment here on this view and the filter.
The filter is fine, but it doesn't apply to the export in calendar.
I wanted to import the contribution to one session (5 talks), and I ended up having all the contribution from the full conference (≥100 ones!).
